libs/zgui/build.zig

```diff
@@ -112,6 +117,16 @@ pub fn build(b: *std.Build) void {
         .flags = cflags,
     });
 
+    if (options.with_freetype) {
+        imgui.addCSourceFile(.{
+            .file = b.path("libs/imgui/misc/freetype/imgui_freetype.cpp"),
+            .flags = cflags,
+        });
+        imgui.defineCMacro("IMGUI_ENABLE_FREETYPE", "1");
+        const freetype = b.dependency("freetype", .{});
```

Comment:
`lazyDependency` would be better here so that freetype is only fetched if the option is enabled.
This reminded me to make this commit: https://github.com/zig-gamedev/zig-gamedev/commit/cf15279a372dcb8d0482bcb6b89851b562c122ed and start making use of lazy dependencies.

Reply:
Yea. I made a local commit that uses lazy dependencies for exactly the reason you mention. But since all the other dependencies were still eager I decided against being the first to break the ice :P